include/fonts/DS_DIGI12pt7b.h

```diff
@@ -0,0 +1,218 @@
+const uint8_t DS_DIGI12pt7bBitmaps[] PROGMEM = {
```

Comment:
What's the license of this font? Just to be sure, please add a `.txt` containing it beside it. We all hate legal stuff :)

Reply:
This font is the downscaled version of the font [DS_DIGI30pt7b.h](https://github.com/Open-Smartwatch/open-smartwatch-os/blob/master/include/fonts/DS_DIGI30pt7b.h) , already used in this project. As far as i know, it is not possible to resize a font using gfx, so i was forced to rescale it directly at the source.( it's not an efficient move ) . Anyway , the license is already [present ](https://github.com/Open-Smartwatch/open-smartwatch-os/blob/23232eb4ab1dd44ad7aa45463de78a7c177c2958/include/fonts/DS_DIGI30pt7b.font.url.txt), but thanks to your comment i figured out that including this font in the project may be a problem. The license is not well defined and i don't know if using this font for the weather app is considered "personal use", probably not. Now i will look for a different font on the adafruit website.

Reply:
I think this is fine :)

I'll add a comment for that :+1: